9-simple-exercises/hello world/src/App.jsx

```diff
@@ -1,5 +1,4 @@
 function App() {
-
   return <h1>Hello world!</h1>;
```

Comment:
You're supposed to render a square which contain hello word text I am only seeing h1